### PR TITLE
Wait up to 10 seconds for the current API rate-limit period to expire…

### PR DESCRIPTION
**Pull request by jasongillis**
… instead of 10 or more seconds.

Thank you for contributing to MythTV.

As many of our developers do not visit GitHub it is important to track
all contributions at our central ticket tracker over at code.mythtv.org

To prevent your contributions being overlooked please create a new ticket
there and refer to this pull request. (Bonus points for linking to the
ticket in the pull request, too. It helps us noticed potentially overlooked
pull requests due to missing tickets.)

https://code.mythtv.org/trac/wiki/TicketHowTo
https://code.mythtv.org/trac/newticket

### CODE REVIEW

**Comment by rcrdnalor** [comment]
The corresponding trac issue is https://code.mythtv.org/trac/ticket/13325
See my comment there. Sorry for not updating this pull request with a back link to the ticket.
TLDR: rate limiting is no more in use by themoviedb.
I kept this issue open because tmdb3.py and its associated cache implements a kind of rate-limiting, too. 

BTW, the new thetvdb.com api used by ttvdbv4.py takes longer to fetch a single entry than tmdb3.py.

**Comment by rcrdnalor** [comment]
I agree with this [comment](https://github.com/MythTV/mythtv/pull/171#issuecomment-1402669139) . PR closed.
